src/paho/mqtt/client.py

```diff
@@ -600,7 +601,8 @@ def __init__(self, client_id="", clean_session=None, userdata=None,
         self._ssl_context = None
         # Only used when SSL context does not have check_hostname attribute
         self._tls_insecure = False
-        self._logger = None
+        logging.basicConfig(level=logging.DEBUG)
+        self._logger = logging.getLogger("plop")
```

Comment:
plop! 😂

Reply:
Thanks, totally missed this temporary debugging code.